frontend/src/data/variables/BrfssProvider.ts

```diff
@@ -159,6 +176,7 @@ class BrfssProvider extends VariableProvider {
     df = df
       .dropSeries([
         "population",
+        "population_pct",
```

Comment:
if you're renaming it, it shouldn't still be there to drop, right?

Reply:
Right, good call